plugins/e57/io/E57Writer.cpp

```diff
@@ -153,6 +165,25 @@ void E57Writer::initialize()
     }
 }
 
+void E57Writer::addDimensions(PointLayoutPtr layout)
+{
+    m_extraDims = std::make_unique<e57plugin::ExtraDims>(e57plugin::parse(m_extraDimsSpec));
```

Comment:
@pravinshinde825 could you please stick to C++11 conventions?

Reply:
done